### DISCUSSION TITLE
Escape To Dismiss

### DISCUSSION THREAD

**Discussion by freshleafmedia** [pboivin/filament-peek#14] [Ideas]
It is expected that modals type UIs are dismissed when the escape key is pressed.

Is this something that could be added?

**Comment by pboivin** [comment]
Yep, thank you for the suggestion! Will add in the next patch release.

**Comment by pboivin** [comment]
Just released v0.2.4 with a fix to close the modal with the escape key when the focus is inside of the iframe. I've tested on multiple browsers but let me know if you run into any other issues. Thanks again :)

https://github.com/pboivin/filament-peek/releases/tag/v0.2.4
